src/main/resources/templates/post-details.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en"
+      xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<head>
+    <title th:text="${post.title}">Post Details</title>
+</head>
+<body>
+<div layout:fragment="content" class="container mt-4">
+    <!-- Título del post -->
```

Comment:
Could you please change the comments to English as others may not be familiar with the language.